deploy/charts/cert-manager/values.yaml

```diff
@@ -195,6 +195,12 @@ webhook:
   # Optional additional annotations to add to the webhook Pods
   # podAnnotations: {}
 
+  # Optional additional annotations to add to the webhook MutatingWebhookConfiguration
+  # mutatingWebhookAnnotations: {}
+
+  # Optional additional annotations to add to the webhook ValidatingWebhookConfiguration
+  # validatingWebhookAnnotations: {}
```

Comment:
Ditto : `validatingWebhookConfigurationAnnotations`

Reply:
Thx. Resolved and force-pushed.

Reply:
Thx. Resolved and force-pushed.